src/Test.ConcurrencyTests/rewrite.coyote.Windows.Debug.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@shaopeng-gh This file (and the corresponding `Release` version) needs to checked in. These files are used by the post build step to figure out which dlls to instrument.